stetho-sample/build.gradle

```diff
@@ -26,11 +26,17 @@ android {
             signingConfig signingConfigs.debug
         }
     }
+
+    lintOptions {
+        // Rhino has references to awt and swing
+        disable 'InvalidPackage'
+    }
 }
 
 dependencies {
     debugCompile project(':stetho')
     debugCompile project(':stetho-urlconnection')
+    debugCompile project(':stetho-repl-js')
```

Comment:
Hmm, might need to think carefully about including this.  Specifically, folks look at stetho-sample as a clue as to how big Stetho will make their APK (~100KB bigger) but with Rhino it will explode.  That could scare off some folks that don't dig into it enough to realize that Rhino is optional.

Furthermore, folks might copy the sample into their project and again not realize why they now hit the dex limit.

Perhaps a separate sample for the repl-js stuff makes sense?